### PR TITLE
Improved server performance

### PR DESCRIPTION
**Pull request by RageCage64**
It was too slow before, this should help

### CODE REVIEW

**Review by jameswklassen — APPROVED** [review]
So clean 😩

**Comment by RageCage64** [comment]
Wow, didn't even give a reason for closing. You've got a lot of work to become an open source maintainer, pal

**Comment by jameswklassen** [comment]
Too bad! 🙁  Hopefully we see some of these changes implemented in the future. Maybe splitting up the PRs would be applicable here. 👍

File: server.py [hunk 1]
```diff
@@ -1,806 +1 @@
-#!/usr/local/bin/python3
```
Comment (NickJosephson) [inline server.py:1]: Should probably leave the shebang in for compatibility. Other than that, this is a solid PR that addresses many issues.

Comment (RageCage64) [reply server.py:1]: Great point, thanks. I am new to Python development, so still getting the hang of all this. 😄

**Pull request closed without merging**